CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project(tango_icons_vendor)
 
 find_package(ament_cmake REQUIRED)
 
+option(INSTALL_TANGO_ICONS "Install Tango icons" OFF)
```

Comment:
Currently the option reads `OFF` on macOS / Windows but still installs the icons. Please update the code so that the default value is selected based on the platforms and the following condition just check the actual value of `INSTALL_TANGO_ICONS`.

Reply:
This option only makes sense on Linux, We must install the icons on MacOS and Windows. That's why the logic remains the same. I can rename this to INSTALL_TANGO_ICONS_ON_LINUX. By the way , the current logic (https://github.com/ros-visualization/qt_gui_core/pull/222) it's going to use the default Qt icons not the Tango ones.

Reply:
> By the way , the current logic (ros-visualization/qt_gui_core#222) it's going to use the default Qt icons not the Tango ones.

With "current logic" do you mean:
* the logic currently used in qt_gui_core (master branch, released versions) or
* the logic change proposed in the referenced PR?

Reply:
the logic change proposed in the referenced PR

Reply:
> This option only makes sense on Linux, We must install the icons on MacOS and Windows.

That is why the option should default to `ON` on macOS and Windows and `OFF` on Linux. But since the option does exist it should be honored on all platforms if the user decides to set it explicitly.

Reply:
https://github.com/ros-visualization/tango_icons_vendor/pull/4/commits/48646539ec7e9abae68bd20c8dbc18d24169150d

Reply:
This doesn't change the *default* on a per platform base but *overrides* the option on some platforms.